asyncapi/json-schemas/operational/assignment/omit/response/assignment-omit-response-no-intention.example.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Assignment Unable Response - NO INTENTION",
+  "payload": {
+    "eventTimeStamp": "2023-04-27T12:40:12.731913Z",
+    "traceId": "083b71c4-e417-11ed-b5ea-0242ac120003",
+    "result":{
+      "code": "OK",
+      "reason": "OK",
+      "description": "The plan has been updated"
```

```suggestion
      "description": "The omission is successfully registered"
```

We are not updating the plan